examples/gfx/opengl/5_opengl_flappy_bird/core/game.cpp

```diff
@@ -18,7 +18,7 @@ namespace game::core {
 
         camera = make_shared<OrthographicCamera>(aspect * -7.0f, aspect * 7.0f, 7.0, -7.0, -7.0f, 7.0f);
  
-        const auto shaderFlags = ENABLE_UNIFORMS;
+        const auto shaderFlags = EnableUniforms;
 
         auto vertexShader = GLShaderFactory::create_shader("assets/vertex.glsl", ShaderTypes::VERTEX_SHADER, shaderFlags);

```

Comment:
forgot these shader variables

Reply:
That's an example file. Example file variables I won't be fixing in this PR, because they are not relevant to it
I will fix the examples in the separate PR's